source/sass/buyers-guide/views/about.scss

```diff
@@ -64,7 +64,7 @@
         bottom: 0;
 
         &:focus {
-          background: red;
+          background: $dark-red;
```

Comment:
kind of the same nit, but I guess we're using a uniform naming scheme that ignores real world colour naming =(